### ISSUE TITLE
beta2

### ISSUE THREAD

**Issue by andrewg78** [BIPES/BIPES#83]
beta2 is hardcoded in several requests
It leads to 404 when BIPES is used in local webserver
![beta2](https://user-images.githubusercontent.com/2042344/147391833-05bd5788-e33d-4c34-8d77-1551852a780f.png)

**Comment by rafaelaroca** [comment]
You are right. Sorry for that and thanks for reporting. We will work on making these paths relative.

**Comment by andrewg78** [comment]
I can rise a pull request on that. I did fix this on my end already.
Not sure about urls to https://bipes.net.br/beta2/ui/
I did not change them, but maybe they should be  relative as well?
![beta2-bipes](https://user-images.githubusercontent.com/2042344/147393316-0e74803b-61b8-43fb-8377-7d9300be1da7.png)

**Comment by JorgeGMarques** [comment]
Yes, every url should be relative! We even have been using another url for a while now (bipes.net.br/**ide**  instead of bipes.net.br/**beta2/ui**).

Feel free to open a pull request :)
